content/integrations/awsdirectconnect.md

```diff
@@ -8,9 +8,9 @@ doclevel: basic
 
 ## Overview
 
-AWS Direct Connect makes it easy to establish a dedicated network connection from your premises to AWS.
+This integration collects metrics from AWS Direct Connect (e.g. `aws.dx.connection_state`, `aws.dx.connection_bps_egress`)
 
-Enable this integration to see in Datadog all your Direct Connect metrics
+Enable this integration to see in Datadog all your Direct Connect metrics.
```

Comment:
Let's omit this sentence. As with any integration, it's obvious that collected metrics can be seen in Datadog.